src/modules/Sidebar/Sidebar.js

```diff
@@ -101,7 +138,7 @@ class Sidebar extends Component {
     const rest = getUnhandledProps(Sidebar, this.props)
     const ElementType = getElementType(Sidebar, this.props)
 
-    return <ElementType {...rest} className={classes}>{children}</ElementType>
+    return <ElementType {...rest} className={classes} ref={(c) => { this.ref = c }}>{children}</ElementType>
```

Comment:
The ref handler needs to be hoisted here.  Otherwise, this creates a new function on every render.
```jsx
handleRef = c => { this.ref = c }

<ElementType ref={this.handleRef} />
```